### DISCUSSION TITLE
How do I change the number of users I can display?

### DISCUSSION THREAD

**Discussion by ghost** [wp-graphql/wp-graphql#1862] [Q&A] [answered]
Currently it only shows 10 users. How do I increase this?

And, how can I change the page I want to show users on eg: see page 1 with 30 users on it, or see page 2 with 30 users on it etc.

```
query MyQuery {
  users {
    nodes {
      lastName
      firstName
      userId
    }
  }
}
```

**Comment by jasonbahl** [comment, marked as answer]
Hey @Jonofat! 

Apologies for taking so long to reply to this! I hope you found your answer elsewhere in the meantime. 

Anyway, for other folks reading this, I thought it would be good to add some information here. 

When querying connections, you can pass arguments to specify how many items you would like from the data. 

In your case, you're asking for `users` so if we wanted to ask for 50 users we could do that like so: 

```graphql
query MyQuery {
  users(first:50) {
    nodes {
      lastName
      firstName
      userId
    }
  }
}
```

The first argument accepts a number, and WPGraphQL will use that to determine how many nodes to return. 

WPGraphQL will cap at a max of 100 items by default. 

So, if you were to pass `first: 500`, you would only get 100 items, and you would then need to make a paginated request to get more than 100. 

I've written an article about making paginated requests, with code examples here: https://www.wpgraphql.com/2020/03/26/forward-and-backward-pagination-with-wpgraphql/
